docs/modules/ROOT/pages/glossary/index.adoc

```diff
@@ -101,5 +98,8 @@ A mechanism to secure the software supply chain by recording events in a user-de
 **Tekton integration testing** +
 A process that uses Tekton tasks to support the setup and execution of dynamic application tests against container images.
 
+**Tekton task** +
```

Comment:
Would it make sense to create a _separate_ section to define all of the Tekton-relative words? If so, we could also put in items like **TaskRun**. These are not items that we are defining, but vocabulary that we are collecting from elsewhere.

Reply:
i was thinking the same thing. yes!

Reply:
Does "step" need more context, either in this definition, or its own glossary term?